ports/espressif/boards/lolin_c3_pico/board.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Scott Shawcroft for Adafruit Industries
```

Comment:
You may want to change this.

Reply:
I'm hesitant to remove prior copyright notices. What should I change here?

Reply:
It's ok to remove. (That's me.)